### PR TITLE
Allow control over containerPort

### PR DESCRIPTION
**Pull request by camaeel**
Allow control over containerPort. Set default value to 8080.

Motivation - by default port is always set to 80. I wanted to control it and change to different value in order to start the container without root.

### CODE REVIEW

**Comment by camaeel** [comment]
Lint check fails due to the default value of DRONE_SERVER_HOST - it is empty string, but it is required to have minimum length of  3.

File: charts/drone/templates/configmap.yaml [hunk 1]
```diff
@@ -9,6 +9,9 @@ data:
     Rather than maintain a comprehensive ConfigMap, we map all sub-keys of the "env" value here.
     This allows for more flexibility and less Chart churn as Drone evolves.
   */}}
-{{- range $envKey, $envVal := .Values.env }}
+{{- $droneServerPort := printf ":%v" .Values.containerPort }}
+{{- $defaultEnvs := dict "DRONE_SERVER_PORT" $droneServerPort }}
+{{- $mergedValues := merge .Values.env $defaultEnvs }}
+{{- range $envKey, $envVal := $mergedValues }}
```
Comment (bradrydzewski) [inline charts/drone/templates/configmap.yaml:15]: please remove

File: charts/drone/values.yaml [hunk 2]
```diff
@@ -13,6 +13,9 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+# Change containerPort to some custom value
+containerPort: 8080
```
Comment (bradrydzewski) [inline charts/drone/values.yaml:17]: the default ports are 80 and 443.  I don't think this should default to 8080

**Comment by bradrydzewski** [comment]
closing in favor of https://github.com/drone/charts/pull/21/files

**Pull request closed without merging**